pom.xml

```diff
@@ -20,6 +20,8 @@
 
         <zeebe.version>8.1.5</zeebe.version>
         <version.zeebe.spring>8.1.12</version.zeebe.spring>
+        <kafka.version>3.1.2</kafka.version>
```

Comment:
Kafka version should be 3.2.+ due to some weird bugs we've seen in other services. Most other services are using 3.2.+